agent/src/main/java/com/microsoft/hydralab/agent/runner/ActionExecutor.java

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+package com.microsoft.hydralab.agent.runner;
+
+import com.alibaba.fastjson.JSONObject;
+import com.microsoft.hydralab.common.entity.common.DeviceAction;
+import com.microsoft.hydralab.common.entity.common.DeviceInfo;
+import com.microsoft.hydralab.common.management.DeviceManager;
+import com.microsoft.hydralab.common.util.ThreadUtils;
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * @author zhoule
+ * @date 12/20/2022
+ */
+
+public class ActionExecutor {
+    /**
+     * the implementation of supported actions should not be overload
+     */
+    private Set<String> actionTypes = Set.of("setProperty", "setDefaultLauncher", "backToHome");
```

Comment:
Could you help add the support for changeSystemSetting and changeGlobalSetting?
![image](https://user-images.githubusercontent.com/8344245/208817001-76ba0ece-4d79-41d3-a9ce-9ba8f2db1620.png)